_ml-commons-plugin/model-serving-framework.md

```diff
@@ -33,6 +33,10 @@ Furthermore, files must be saved as zip files before upload. Therefore, to ensur
 
 Most deep learning models are over 100 MBs, making it difficult to fit the model into a single document. OpenSearch splits the model file into smaller chunks to be stored in a model index. When allocating machine learning (ML) or data nodes for your OpenSearch cluster, be aware of the size of your model to prevent any downtime when making inferences. 
```

Comment:
The last sentence has some grammatical errors. People make inferences but systems perform ML inference. "Ensure that your ML Nodes are properly sized and have enough memory to serve your models for ML inference."